source/include/common/cs_Types.h

```diff
@@ -350,6 +348,20 @@ enum class CS_TYPE: uint16_t {
 	EVT_MICROAPP_ERASE_RESULT,                        // Microapp has been erase from flash, or failed to do so.
 	CMD_MICROAPP_ADVERTISE,                           // A microapp wants to advertise something.
 
+	CMD_BLE_CENTRAL_CONNECT,                          // Connect to a device.    See BleCentral::connect().
```

Comment:
I'm a little concerned what happens if several components would like to concurrently connect to other crownstones (or the same). How will this be 'synchronized'?

Also: if we have a static instance, are we still supposed to use these events?
In that case can we make the `getInstance` function private? (perhaps with CrownstoneClass as friend to initalize it?)

Reply:
There is no synchronization, they will just get a 'busy' or 'already connected' error, and have to try again later.

Currently it's static mostly because of the discovery event handler:
```
void handle_discovery(ble_db_discovery_evt_t* event) {
	// No need for null pointer check.
	BleCentral::getInstance().onDiscoveryEvent(*event);
}
```